python/.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,19 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
The files from .idea folder should not be committed, since it's personal configuration for local text editor